helios-system-tests/src/main/java/com/spotify/helios/system/MasterAuthenticationTest.java

```diff
@@ -76,8 +92,8 @@ public String schemeName() {
     }
 
     @Override
-    public ServerAuthentication<String> serverAuthentication() {
-      return new ServerAuthentication<String>() {
+    public ServerAuthentication<String> serverAuthentication(Map<String, String> environment) {
+      return new SimpleServerAuthentication<String>() {
         @Override
         public Authenticator<String> authenticator() {
           return new Authenticator<String>() {
```

Comment:
Is the semicolon here https://github.com/spotify/helios/pull/713/files#diff-7f9fb5083fc38f934dca424ad96be781R111 required?